src/Adapter/Product/Pack/Update/ProductPackUpdater.php

```diff
@@ -40,6 +40,7 @@
 use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductType;
 use PrestaShop\PrestaShop\Core\Exception\CoreException;
 use PrestaShopException;
+use Product;
```

Comment:
A `use` all by itself?

Reply:
I don't understand .... ProductPackUpdater use the Product class .... nothing strange here ? no ?

Reply:
Not used .... i remove immediatly. Mais be a rule about unused import can help with this.